packages/webdriver/src/bidi/core.ts

```diff
@@ -17,9 +19,10 @@ const RESPONSE_TIMEOUT = 1000 * 60
 
 export class BidiCore {
     #id = 0
-    #ws: WebSocket
+    #ws: WebSocket | null = null
```

Comment:
I am trying to avoid using `null` as much as possible. 
```suggestion
    #ws?: WebSocket
```

Reply:
changed